code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -116,7 +118,7 @@
 				return
 
 		if(QUALITY_PRYING)
-			if(!locked)
+			if(locked != TRUE)
```

Comment:
Locked is a ternary value, true/false has no meaning and is misleading. Should be 1 or a defined constant.